beacon-chain/sync/subscriber_handlers.go

```diff
@@ -2,26 +2,50 @@ package sync
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/gogo/protobuf/proto"
 	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/go-ssz"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
 )
 
 func (r *Service) voluntaryExitSubscriber(ctx context.Context, msg proto.Message) error {
+	ve, ok := msg.(*ethpb.SignedVoluntaryExit)
+	if !ok {
+		return fmt.Errorf("wrong type, expected: *ethpb.SignedVoluntaryExit got: %T", msg)
+	}
 	s, err := r.chain.HeadState(ctx)
 	if err != nil {
 		return err
 	}
-	r.exitPool.InsertVoluntaryExit(ctx, s, msg.(*ethpb.SignedVoluntaryExit))
+	r.exitPool.InsertVoluntaryExit(ctx, s, ve)
 	return nil
 }
 
 func (r *Service) attesterSlashingSubscriber(ctx context.Context, msg proto.Message) error {
-	// TODO(#3259): Requires handlers in operations service to be implemented.
+	as, ok := msg.(*ethpb.AttesterSlashing)
+	if !ok {
+		return fmt.Errorf("wrong type, expected: *ethpb.AttesterSlashing got: %T", msg)
+	}
+	s, err := r.db.State(ctx, bytesutil.ToBytes32(as.Attestation_1.Data.BeaconBlockRoot))
```

Comment:
Somebody can spam panic nodes by sending us slashings that have these nil inner fields, fixing